components/form/FormItem.razor

```diff
@@ -6,8 +6,9 @@
     @if (!string.IsNullOrEmpty(Label))
     {
         <AntDesign.Col @attributes="GetLabelColAttributes()" class=@($"{_prefixCls}-label")>
-            <label class=@(Required ? $"{_prefixCls}-required" : _labelCls)> @(Label)</label>
-        </AntDesign.Col>
+        <label class=@(Required ? $"{_prefixCls}-required" : _labelCls)> @(Label)</label>
+        </AntDesign.
```

Comment:
Would seem like the formatter is breaking some of the Razor pages, will investigate this

Reply:
ReSharper is absolute shit when it comes to formatting Razor files. Is there a way you can limit it to .cs files?

Reply:
That's a shame, I chose ReSharper for it's ability to format Razor files :( Guess I will investigate using dotnet format instead